### DISCUSSION TITLE
Next js Image Component sizes

### DISCUSSION THREAD

**Discussion by j-o-o** [vercel/next.js#19814] [Help]
I'm uploading photos from my phone to a CMS and later on display it on my website.
Now I want to use the Next Image component to convert them into smaller images.
I just don't get the size property. Some of the images are smaller. So if I say I want to display a 10.000 px width image , I go `width={image.width / 10}` (while `image.width` is taken from the meta data) to scale it to 1.000 px. But is the image is smaller, it scales down smaller. (e.g. it results in the image being just 200px width) 
A maxWidth property would be cool for that, so that the already small images are not affected by the division. Can someone provide me with a hint on how to achieve this?

Every image needs to be 600px width, while the dimensions are the same.

**Comment by j-o-o** [comment]
I solved this problem with a container:

```
                  <Container
                    style={{
                      position: "relative",
                      width: "auto",
                      maxWidth: "500px",
                      minwidth: "200px",

                      height: "auto",
                      maxHeight: "500px",
                      minHeight: "300px",
                    }}
                  >
                    <Image
                      src={image.url}
                      objectFit="contain"
                      objectPosition="left"
                      layout="fill"
                      quality={60}
                      style={{ float: "right" }}
                      title={title}
                    />
                  </Container>
```

But how do I resize the images to smaller files? It's so confusing. If I set width and height attributes, the component wants me to remove `layout="fill"` arghhhhhhhh

**Reply by nghiepnguyen520** [reply]
Image of Next was not optimized resize. Don't match the document of the next intro before
